manifests/charts/base/templates/endpoints.yaml

```diff
@@ -1,23 +1,21 @@
 {{- if .Values.global.remotePilotAddress }}
-  {{- if not .Values.global.istiod.enabled }}
+  {{- if .Values.global.pilot.enabled }}
```

Comment:
Pilot ? Why ?

Reply:
Actually looking at this close, this should be `.Values.pilot.enabled` not `.Values.global.pilot.enabled` - the former already exist and controls whether or not we will deploy istiod. It should be `istiod` not `pilot` but I am not sure if its worth breaking the "API"?

Reply:
https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/23208/integ-multicluster-k8s-tests_istio/3173/build-log.txt

This breaks the multicluster/remote test:
```
Error rendering the manifest: template: base/templates/services.yaml:2:16: executing "base/templates/services.yaml" at <.Values.global.pilot.enabled>: nil pointer evaluating interface {}.enabled
```

Reply:
Yeah I am surprised this passes the linter...